app/decorators/ncr/work_order_decorator.rb

```diff
@@ -2,27 +2,27 @@ module Ncr
   class WorkOrderDecorator < Draper::Decorator
     delegate_all
 
-    EMERGENCY_APPROVER_EMAIL = 'Emergency - Verbal Approval'
-    NO_APPROVER_FOUND = 'No Approver Found'
+    EMERGENCY_APPROVER_EMAIL = "Emergency - Verbal Approval"
+    NO_APPROVER_FOUND = "No Approver Found"
 
     def current_approver_email_address
-      approver_email_address(current_approver)
-    end
-
-    def final_approver_email_address
-      approver_email_address(final_approver)
-    end
-
-    def status_aware_approver_email_address
       if proposal.approved?
         final_approver_email_address
       else
-        current_approver_email_address
+        pending_approver_email_address
       end
     end
 
     private
 
+    def final_approver_email_address
+      approver_email_address(final_approver)
+    end
+
+    def pending_approver_email_address
```

Comment:
is the use of `pending` here meant to correlate with the step status? Does `pending` == `current` ?